kastle/connection.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+from typing import Dict
+
+from . import errors
+
+CRLN = b"\r\n"
```

Comment:
```suggestion
from . import errors


CRLN = b"\r\n"
```
Please, add one more blank-line to follow PEP-8 conventions.

Reply:
This follow the PEP-8. The two blank lines is for functions and classes.